lib/gitx/cli/integrate_command.rb

```diff
@@ -20,7 +20,7 @@ def integrate(integration_branch = 'staging')
         begin
           execute_command(UpdateCommand, :update)
         rescue
-          fail MergeError, 'Merge Conflict Occurred. Please Merge Conflict Occurred. Please fix merge conflict and rerun the integrate command'
+          raise MergeError, 'Merge Conflict Occurred. Please Merge Conflict Occurred. Please fix merge conflict and rerun the integrate command'
```

Comment:
Line is too long. [144/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.